addon/ng2/tasks/npm-install.js

```diff
@@ -0,0 +1,17 @@
+/* jshint node: true, esversion: 6 */
```

Comment:
Yes, I saw, I will prepare the demo which is requested in the issue.

I am pretty sure it is not the issue on our side. The initial `ng install` used that task and the were Windows users complaining about it (including @filipesilva). I will also test it myself on the VM as you suggested.